scripts/genpinctrl/stm32f1-pinctrl-config.yaml

```diff
@@ -256,6 +256,10 @@
   match: "^USB_OTG_FS_(?:DM)?(?:DP)?(?:SOF)?(?:ID)?(?:VBUS)?$"
   mode: input
 
+- name: JTAG PORT
+  match: ^(SYS|DEBUG)_((JTMS-)?SWDIO|(JTCK-)?SWCLK|JTDI|JTDO(-TRACESWO|-SWO)?|(NJ)?JTRST)$
+  mode: alternate
+
```

Comment:
Nit: Move it just after I2S to keep alphabetical order

Reply:
This is done on top of the branch. You just need to rebase on top of https://github.com/zephyrproject-rtos/hal_stm32/commit/37842371f5ef0078ad32f16e5059c1df58b51892